client/asyncns.c

```diff
@@ -77,7 +77,7 @@ enum {
 };
 
 struct asyncns {
-    int fds[MESSAGE_FD_MAX];
+    int fds[4];
```

Comment:
Hm, but if we update this code next time from upstream, how can we know what to reapply? Maybe we should have some patch and apply it during our compilation? Or we should try to get those things upstream.